state_machine/applications/flight/Tasks/radio.py

```diff
@@ -64,29 +66,29 @@ async def main_task(self):
                 self.debug(f'Recieved msg "{response}", RSSI: {cubesat.radio.last_rssi - 137}')
 
                 if header == headers.NAIVE_START or header == headers.NAIVE_MID or header == headers.NAIVE_END:
```

Comment:
These should be changed to something like `MEMORY_BUFFERED_START`, `_MID` and `_END`